packages/contracts/deploy/010-AddressDictator.deploy.ts

```diff
@@ -76,7 +66,7 @@ const deployFn: DeployFunction = async (hre) => {
 
   await deployAndPostDeploy({
     hre,
-    name: 'AddressDictator',
+    name: unmanagedNames.addressDictator,
     contract: 'AddressDictator',
```

Comment:
I know this diff looks a bit odd: why would you only replace one of the two identical strings?

Ideally the two arguments would have different types. This [SE answer](https://stackoverflow.com/a/45801281/7230551) points at a decent way, but I think this is good enough for now.

Reply:
Actually, when `name` and `contract` are the same, we can leave out `contract` entirely. So you could delete the `contract` parameter here.